src/components/AR/frameworks/xr8/ixr8-ui-handler.ts

```diff
@@ -0,0 +1,10 @@
+interface IXR8UIHandler {
```

Comment:
Thjis could be exported in the main file that uses it right? Dont think we need a file just for those few lines

Reply:
The user can create a custom XR8UIhandler. It would look strange if this would be exported from xr8-provider

Reply:
If you re-export it that's fine, in the `index.js` file